src/App.js

```diff
@@ -1,10 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-3 :+1: